### ISSUE TITLE
Group sale notifications into one email

### ISSUE THREAD

**Issue by CGeorges** [CGeorges/unrealsales-tracker#6]

**Comment by thsxthz** [comment]
Yes please this.  All the notifications I get everyday are making my inbox unusable.  I literally am getting something like over a thousand emails every week or so now the vast majority of which come from here.  I have to spend way to much time digging out actually important things from the chaff.  And I can't actually tell which which sales are of importance because of the flood of so many notices at once all the time.  Also, how about simply the option to turn off emails wholesale until this collating function is introduced, and eventually longer term the ability to select individual items to be notified, or not notified about by email when they go on sale.
